### PR TITLE
Creation of control variables tables

### PR DESCRIPTION
**Pull request by RemiCSK**
Contains CODGEO, LIBGEO, first quartile available income, median, 3 quartile,Gini index (for 2014 and 2019),population density, physicists accessibility (only for 2019, 2014 not available)

Missing criminality, will commit later

### CODE REVIEW

File: src/controlvariables.py
Comment (pwisEcon) [inline src/controlvariables.py:1]: Seems good!

**Review by pwisniewski-econ — APPROVED** [review]
Good job! Everything looks good! I just added a conversion to long format for unemployment controls.
 

unemp

**Comment by RemiCSK** [comment]
> Contains CODGEO, LIBGEO, first quartile available income, median, 3 quartile,Gini index (for 2014 and 2019),population density, physicists accessibility (only for 2019, 2014 not available)
> 
> Missing criminality, will commit later



> Good job! Everything looks good! I just added a conversion to long format for unemployment controls.
> 
> unemp

Looks good 👍

**Review by lu-julia — APPROVED** [review]
Well done! Ready to merge!

**Pull request merged**